oid-info.md

```diff
@@ -0,0 +1,28 @@
+# Rekor OID Information
+
+## Description
+
+This document defines Rekor
+[OID values](https://github.com/sigstore/sigstore/blob/main/docs/oid-info.md).
+
+Rekor reserves the `1.3.6.1.4.1.57264.3` OID root for all of its values.
+
+## Directory
+
+| OID                       | Name                   | Description                                                                                                   |
+| ------------------------- | ---------------------- | ------------------------------------------------------------------------------------------------------------- |
+| 1.3.6.1.4.1.57264.3.1     | Integrated Time        | When the data was added to the log.                                                                           |
+| 1.3.6.1.4.1.57264.3.2     | Log ID                 | This is a SHA256 hash of the DER-encoded public key for the log at the time the entry was included in the log |
+| 1.3.6.1.4.1.57264.3.3     | Log Index              | The index of the entry in the transparency log.                                                               |
+| 1.3.6.1.4.1.57264.3.4     | Verification           | Log Entry Verification data.                                                                                  |
+| 1.3.6.1.4.1.57264.3.4.1   | Inclusion Proof        | Proof of inclusion on the transparency log.                                                                   |
```

Comment:
Same question here, is anything set for this OID?

Reply:
Nope.